src/app/shared/data-storage.service.ts

```diff
@@ -46,8 +54,13 @@ export class DataStorage{
                 if(this.userData?.password){
                     if(this.userData.password === password){
                         this.loggedInUser.next(this.userData);
-                        this.router.navigate(['/profile',this.userData.id]);
-                        this.isAuth.next(true);
+                        if(this.userData.userId === "ADMINE"){
```

Comment:
ADMINE??

Reply:
Admin component is yet to be decided. So ignoring this comment.

Reply:
spelling.